### PR TITLE
(GH-10) Added TwitterStats

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
Fixes #10

File: TwitterStats/run.ps1 [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                    $AuthParams.Add('HttpEndPoint', $ResourceURL)
+                    $AuthParams.Add('RESTVerb', $RestVerb)
+                    $AuthParams.Add('Params', $Parameters)
+                    $AuthParams.Add('OAuthSettings', $OAuthSettings)
+                    $AuthorizationString = Get-OAuth -AuthorizationParams $AuthParams                 
+                    $HTTPEndpoint= $ResourceURL
+                    if($Parameters.Count -gt 0)
+                    {
+                        $HTTPEndpoint = $HTTPEndpoint + '?'
+                        $Parameters.Keys | % { $HTTPEndpoint = $HTTPEndpoint + $_  +'='+ [System.Net.WebUtility]::UrlEncode($Parameters.Item($_)).Replace('+','%20') + '&'}
```
Comment (devlead) [inline TwitterStats/run.ps1]: `[System.Net.WebUtility]::UrlEncode` can be replaced with `[Uri]::EscapeDataString`, then no replace is needed and also ensures compat with PowerShell core.

File: TwitterStats/run.ps1 [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+"Begin $twitterData $(Get-Date -Format 'yyyy-MM-dd HH:mm:ss')"
+$partitionKey           = [Guid]::NewGuid().ToString("n")
+$twitterData            = Invoke-TwitterRestMethod -ResourceURL 'https://api.twitter.com/1.1/users/lookup.json' -RestVerb 'GET' -Parameters @{'screen_name' = $twitterAccountsString } -OAuthSettings $OAuth `
+                            | ForEach-Object { $_ } `
+                                | ForEach-Object {
+                                    [PSCustomObject][Ordered]@{
+                                        'FileType'          = 'twitter'
+                                        'RowKey'            = $_.id_str
+                                        'PartitionKey'      = [Guid]::NewGuid().ToString("n")
```
Comment (devlead) [inline TwitterStats/run.ps1]: Should use `$partitionKey`

**Review by devlead — CHANGES_REQUESTED** [review]

File: TwitterStats/run.ps1 (hunk 1 above)
Comment (gep13) [reply TwitterStats/run.ps1]: @devlead good catch.  I borrowed this function directly.  I have made the suggested change.

File: TwitterStats/run.ps1 (hunk 2 above)
Comment (gep13) [reply TwitterStats/run.ps1]: Done.

**Review by devlead — APPROVED** [review]

**Pull request merged**